README.md

```diff
@@ -126,6 +126,8 @@ const user = User.fromJSON(`{ "name": "Beth"}`)
 
 By default `fromJSON` serializes only keys that have been defined in the entity. If you want to add other keys during serialization, use `.fromJSON(data, { allowExtraKeys: true })`.
 
+If you use fields with the default value, the default value continues if inside the function fromJSON does not have the new value for this variable
```

Comment:
Suggested text:
Default field values will be applied for keys not present in `value`.